### ISSUE TITLE
singer-infer-schema is missing a main method

### ISSUE THREAD

**Issue by kamal** [singer-io/singer-tools#1]
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2299, in resolve
    return functools.reduce(getattr, self.attrs, module)
AttributeError: module 'singertools.infer_schema' has no attribute 'main'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/singer-infer-schema", line 11, in <module>
    load_entry_point('singer-tools==0.0.1', 'console_scripts', 'singer-infer-schema')()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 561, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2291, in load
    return self.resolve()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2301, in resolve
    raise ImportError(str(exc))
ImportError: module 'singertools.infer_schema' has no attribute 'main'
```

**Comment by mdelaurentis** [comment]
Thanks for the report @kamal. Can you tell me what commands you ran to install and run `singer-infer-schema`?

I tried to reproduce the error locally and I realized that we never actually published this artifact to PyPi. I just did that, and I was able to `pip3 install singer-tools` and then run `singer-infer-schema`. Now that we've published an artifact to PyPi would you mind trying to run it that way?

**Comment by kamal** [comment]
I ran `python3 setup.py install` from inside the singer-tools repo. I've removed all the files and installed it via `pip3 install singer-tools`. However, here's the result

```
Kamals-MacBook-Pro:tap-clubhouse kamal$ singer-infer-schema < data.json 
{
  "type": [
    "null"
  ]
}
Traceback (most recent call last):
  File "/usr/local/bin/singer-infer-schema", line 7, in <module>
    from singertools.infer_schema import main
ImportError: cannot import name 'main'
```

**Comment by mdelaurentis** [comment]
I believe I fixed the issue. Sorry about that. We wrote this initially as an ad-hoc script and when I moved it into this repository apparently I forgot to wrap it in a main function. I reproduced the issue you found, fixed it, and then I just tried installing it with pip and running it on some real data and it worked.

Fixed in version 0.0.2. Can you confirm whether it's working for you now?

**Comment by mdelaurentis** [comment]
I believe this is fixed now. Closing.
